src/local-editor-service.ts

```diff
@@ -150,8 +150,18 @@ export class LocalEditorService extends EditorService {
       autocompletion += ` $${tabindex++}`;
     }
     autocompletion += `>`;
-    // TODO(timvdlippe): once insertionPoints are implemented in analyzer, expand
-    // snippet with autocompletion for slots
+    if (e.slots.length === 1 && !e.slots[0].name) {
+      autocompletion += `$${tabindex++}`;
+    } else {
+      for (const slot of e.slots) {
+        const tagTabIndex = tabindex++;
+        const slotName = slot.name ? ` slot=${slot.name}` : '';
```

Comment:
Recommend naming this `slotAttribute` rather than slotName. Also the style that we've recommended to people when writing HTML attributes is to surround them with quotes, so we should autocomplete in that style too:

```typescript
` slot="${slot.name}"`
```